mage_ai/ai/generator.py

```diff
@@ -22,5 +22,11 @@ def generate(self, use_case: LLMUseCase, request: Dict) -> Dict:
             return LLMPipelineWizard().generate_pipeline_documentation(
                 pipeline_uuid,
             )
+        elif use_case == LLMUseCase.GENERATE_BLOCK_WITH_DESCRIPTION:
+            from mage_ai.ai.llm_pipeline_wizard import LLMPipelineWizard
+
+            return LLMPipelineWizard().generate_block_with_description(
```

Comment:
Can you rebase master and make this method call async as well?

Reply:
Made the function to be async